proto/viam/app/data/v1/data.proto

```diff
@@ -57,8 +57,7 @@ message CaptureMetadata {
   map<string, google.protobuf.Any> method_parameters = 11;
   repeated string tags = 12;
   string mime_type = 13;
-  string file_name = 14;
-  string file_ext = 15;
+  string file_ext = 14;
```

Comment:
fyi for this breaking change: this was just added and this proto is what we're returning on export, not what's being stored in the db